homeassistant/bootstrap.py

```diff
@@ -261,25 +314,68 @@ def from_config_dict(config: Dict[str, Any],
             hass.config.config_dir = config_dir
             mount_local_lib_path(config_dir)
 
+    # async loop allready running
```

Comment:
I need a future for return value or exception. Without is not easy to find the trouble if will not work and on failover, hass is None...